translations/en.json

```diff
@@ -110,6 +110,12 @@
   "map.select.layer": "Select the layer to display",
   "nav.back": "Back",
   "next": "next",
+  "organisation.sort.intro": "The organisations are...",
```

Comment:
Maybe having an actual text here would be nice, otherwise a vanilla install of the datahub will look a bit weird :)

Reply:
Good point, what do you think of: `The organisations are the providers of metadata records and their corresponding data. They may be owners of metadata catalogs harvested by this instance or owners of (meta-)data referenced by another catalog.` ?

Reply:
Yes that'll do, we can revisit that later on if we have more ideas!